<?xml version="1.0"?>
<launch>
  <include file="$(find rob_auto_cal)/launch/camera1394_launch.launch" />
  <node pkg="rob_auto_cal" name="chess_features" type="chess_features" args="0.02393"/>
  <include file="$(find ur_modern_driver)/launch/ur10_bringup.launch">
    <arg name="robot_ip" value="192.38.66.226" />    	    
  </include>

   
  <node name="chessboard_move_ur10" pkg="rob_auto_cal" type="chessboard_move_ur10" output="screen" args="/misc/shome/ex09/Thesis/X_file" />


</launch>